Metro-Records/la-metro-councilmatic
# Use environment variable to authenticate to Google Drive

## Overview

This PR transitions the app to use an environment variable to authenticate to Google Drive rather than the current JSON file.

Connects Metro-Records/la-metro-dashboard#123 #1043

## Testing Instructions

 * Decrypt `configs/lametro_service_acct_key.json` and copy the contents to `GOOGLE_SERVICE_ACCT_API_KEY` in `.env.local` (you'll have to move off this branch briefly to do + remove newline characters from the JSON string)
 * Set the `REMOTE_ANALYTICS_FOLDER` value in `.env.local` (you can use the value set in the staging app's Heroku env vars)
 * Run `python manage.py generate_tag_analytics`
 * Verify that the analytics file has been uploaded
